applications/RomApplication/python_scripts/rom_manager.py

```diff
@@ -365,6 +413,54 @@ def LaunchTestHROM(self, mu_test):
         return SnapshotsMatrix
 
 
+    def LaunchRunFOM(self, mu_run):
```

Comment:
```suggestion
    def __LaunchRunFOM(self, mu_run):
```
As far as I see these can be mangled.